_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -46,7 +51,7 @@ There are no major accessibility support issues known for this rule.
 
 ## Background
 
-By setting the `tabindex` [attribute value][] of an `iframe` element to `-1` or some other negative number, it becomes impossible to move the focus into the [browsing context][nested browsing context] of the `iframe` element. Even though its content is still included in the [sequential focus navigation][], there is no way to move the focus to any of the items in the `iframe` using standard keyboard navigation.
+Inside a page, each nested document (mostly, content of an `iframe` element) has its own [tabindex-ordered focus navigation scope][] which is the local tab-order inside this document. Then, all of these are grouped together in a [flattened tabindex-ordered focus navigation scope][] which is effectively the tab-order of the full page. By setting the `tabindex` attribute of a [focus navigation scope owner][] (mostly, an `iframe` element) to a negative value, it is excluded from its containing document's [tabindex-ordered focus navigation scope][]; and therefore its own [tabindex-ordered focus navigation scope][] is excluded from the global [flattened tabindex-ordered focus navigation scope][]. That is, a `button` may be in tab-order of a document, but if that document is included in another page via an `iframe` with negative `tabindex`, then the `button` is not in the tab-order of that page; and therefore not keyboard accessible.
```

Comment:
This text is far more complicated than it needs to be to explain the problem; Tabindex=-1 on iframes means things in the iframe cannot be focused with keyboard navigation.

Reply:
To be discussed in the ACT-R meeting